flink-cdc-e2e-tests/flink-cdc-source-e2e-tests/src/test/java/com/ververica/cdc/connectors/tests/utils/FlinkContainerTestEnvironment.java

```diff
@@ -174,6 +177,33 @@ public void after() {
         mysqlInventoryDatabase.dropDatabase();
     }
 
+    @AfterClass
+    public static void afterClass() {
+        // list all images and remove the ones that are not flink images and mysql image
```

Comment:
Images with prefix `testcontainers` also need to be excluded.

Reply:
Fine, add the condition. The docker image of testcontainers just 12MB.

Reply:
I mean exclude it to keep it not be deleted here, or else I'm not sure if there will be any exception occurs as it should  still be in use. By the way, should the filters be combined with `||` or `&&`?

Reply:
maybe use &&

Reply:
Great catch for this